docs/index.html

```diff
@@ -179,8 +204,8 @@
     </article>
     <article><h1>Flexibility</h1>
         <p>Nakadi instances are stateless. They can be run on AWS with auto-scaling.</p>
-        <p>Using Timelines it is easy to move the traffic to another cluster without moving the data and any service
-            degradation.</p>
+        <p>Using Timelines it, is easy to move the traffic to another cluster without moving the data,
```

Comment:
I think first coma is in a wrong place and I think comas are not needed in this sentence at all.